src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private String name = "Isosceles Trapezoid";
+    private String color;
+    private double firstLeg, secondLeg, thirdLeg, height, smallPiece;
+
+    public IsoscelesTrapezoid(double firstLeg, double secondLeg, double thirdLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.thirdLeg = thirdLeg;
+        this.color = color;
+    }
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. Also, the height is not passed to the constructor but is used in the 'findArea' method. You should either calculate it within the constructor or have a method to calculate and set it before 'findArea' is called.